monitoring/bin/deploy_monitoring_tenant_openshift.sh

```diff
@@ -0,0 +1,258 @@
+#! /bin/bash
+
+# Copyright © 2021, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cd "$(dirname $BASH_SOURCE)/../.."
+source monitoring/bin/common.sh
+
+if [ "$OPENSHIFT_TENANT_MONITORING_ENABLE" != "true" ]; then
+  log_error "OpenShift tenant monitoring is currently under development and is not supported"
+  exit 1
+fi
+
+if [ "$OPENSHIFT_CLUSTER" != "true" ]; then
+  if [ "${CHECK_OPENSHIFT_CLUSTER:-true}" == "true" ]; then
+    log_error "This script should only be run on OpenShift clusters"
```

Comment:
Reword to make it clear that it appears that the script is being run on a non-OpenShift cluster.